components/script/dom/webglrenderingcontext.rs

```diff
@@ -2031,6 +2041,15 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             return self.webgl_error(InvalidEnum);
         }
 
+        // The only time that drawingBufferWidth/drawingBufferHeight will change values is:
+        // * the canvas is resized
+        // * an opaque framebuffer is bound
+        // * the default framebuffer is bound, replacing an opaque framebuffer
```

Comment:
Hmm, looking at https://www.khronos.org/registry/webgl/specs/latest/1.0/#DOM-WebGLRenderingContext-drawingBufferWidth it's not obvious that attaching an opaque frame buffer should change the drawingBufferWidth/Height. Perhaps those should always be the size of the default framebuffer? So resizing should be the only thing that changes them, and we might not need any IPC to get their value?